app/assets/stylesheets/partials/_header.scss

```diff
@@ -1,118 +0,0 @@
-header {
```

Comment:
If possible, renaming is cleaner to do with `git mv` instead of a raw rename so that git doesn't think that a file was completely destroyed and created.